proto/cosmos/crisis/crisis.proto

```diff
@@ -13,3 +14,14 @@ message MsgVerifyInvariant {
   string invariant_module_name = 2 [(gogoproto.moretags) = "yaml:\"invariant_module_name\""];
   string invariant_route       = 3 [(gogoproto.moretags) = "yaml:\"invariant_route\""];
 }
+
+// genesis types
+
+// GenesisState - crisis genesis state
+message GenesisState {
```

Comment:
even though it's only 6 lines, I would still put it in a separate genesis.proto, for consistency